android/java/org/chromium/chrome/browser/ntp_background_images/util/NTPUtil.java

```diff
@@ -170,11 +170,16 @@ public static void showNonDistruptiveBanner(ChromeActivity chromeActivity, View
         nonDistruptiveBannerLayout.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                // clickOnBottomBanner(chromeActivity, ntpType, nonDistruptiveBannerLayout, sponsoredTab, newTabPageListener);
-                if (BraveActivity.getBraveActivity() != null) {
-                    nonDistruptiveBannerLayout.setVisibility(View.GONE);
-                    BraveRewardsHelper.setShowBraveRewardsOnboarding(true);
-                    BraveActivity.getBraveActivity().openRewardsPanel();
+                if (BraveAdsNativeHelper.nativeIsBraveAdsEnabled(
+                            Profile.getLastUsedRegularProfile())) {
+                    clickOnBottomBanner(chromeActivity, ntpType, nonDistruptiveBannerLayout,
```

Comment:
Nit: Seems like `nonDistruptiveBannerLayout` has a typo (extra 't' in `Disruptive`)?